packages/client/hmi-client/src/workflow/ops/calibrate-julia/calibrate-operation.ts

```diff
@@ -79,3 +82,18 @@ export const CalibrationOperationJulia: Operation = {
 		return init;
 	}
 };
+
+const runResults = ref<RunResults>({});
+const selectedRun = ref();
+const currentIntermediateVals = ref<{ [key: string]: any }>({ timesteps: [], solData: {} });
+const parameterResult = ref<{ [index: string]: number }>();
+const showSpinner = ref(false);
+const runInProgress = ref<string>();
+export const useJuliaCalibrateOptions = () => ({
+	runResults,
+	selectedRun,
+	currentIntermediateVals,
+	parameterResult,
+	showSpinner,
+	runInProgress
+});
```

Comment:
I don't this will work if you have several Julia-Calibrate operations open, the variables might clobber each other.

Reply:
removed